src/main/java/org/folio/inventory/resources/AbstractInstances.java

```diff
@@ -343,6 +341,16 @@ protected JsonObject toRepresentation(Instance instance,
     putIfNotNull(resp, Instance.METADATA_KEY, instance.getMetadata());
     putIfNotNull(resp, Instance.TAGS_KEY, new JsonObject().put(Instance.TAG_LIST_KEY, new JsonArray(instance.getTags())));
     putIfNotNull(resp, Instance.NATURE_OF_CONTENT_TERM_IDS_KEY, instance.getNatureOfContentIds());
+    if (precedingTitles != null) {
+      JsonArray precedingTitlesJsons = new JsonArray();
+      precedingTitles.forEach(title -> precedingTitlesJsons.add(title.toPrecedingTitleJson()));
+      resp.put(Instance.PRECEDING_TITLES_KEY, precedingTitlesJsons);
+    }
+    if (succeedingTitles != null) {
+      JsonArray succeedingTitlesJsons = new JsonArray();
+      succeedingTitles.forEach(title -> succeedingTitlesJsons.add(title.toSucceedingTitleJson()));
+      resp.put(Instance.SUCCEEDING_TITLES_KEY, succeedingTitlesJsons);
+    }
```

Comment:
Personally, I might put blank lines before each of the if statements to create a visual separation between them and the surrounding code.